src/_gettsim_tests/test_interface.py

```diff
@@ -46,49 +42,20 @@ def func_before_partial(arg_1, arbeitsl_geld_2_params):
 
 
 @pytest.mark.parametrize(
```

Comment:
Do we still need the parametrization here? I find it rather hard to see through what is happening; seems like there is a single case left? But see my request for another test above, maybe that'll bring the usefulness back.